### ISSUE TITLE
Tests started hanging randomly after upgrading to Chrome 130 + TestCafe 3.7.0-rc.1 

### ISSUE THREAD

**Issue by thomas-jakemeyn** [DevExpress/testcafe#8309]
### What is your Scenario?

We run 170+ TestCafe tests in Chrome headless within Docker. They used to be executed reliably in about 15 minutes in Chrome 129. After the upgrade to Chrome 130 + TestCafe 3.7.0-rc.1 (which solves #8286), the same tests now execute in about 30 minutes. After looking at the logs, we realised that some of them (but not always the same) seem to hang randomly.

Note that we don't use the `spec` reporter (see #8307). Also, we pass the following command line switches to Chrome:
```
--disable-fre --no-default-browser-check --no-first-run --disable-search-engine-choice-screen
```

### What is the Current behavior?

Tests hang randomly in Chrome 130 whereas they used to be executed reliably in Chrome 129.

### What is the Expected behavior?

Tests should not hang randomly in Chrome 130.

### What is the public URL of the test page? (attach your complete example)

NA

### What is your TestCafe test code?

NA

### Steps to Reproduce

NA

### TestCafe version

3.7.0-rc.1

### Command-line arguments

NA

### Browser name(s) and version(s)

Chrome 130

### Platform(s) and version(s)

Ubuntu 24.04 (Docker)

**Comment by thomas-jakemeyn** [comment]
I just checked usage of CPU, disk and memory during test execution and everything seems perfectly fine. So the problem is apparently not related to a lack of resources.

![Screenshot 2024-10-22 at 10 50 15](https://github.com/user-attachments/assets/4f66fbb7-c8b5-4147-a041-162fc97bc05f)

**Comment by thomas-jakemeyn** [comment]
I tested with Chrome 130 + TestCafe 3.6.2 patched with a backport of [this change](https://github.com/DevExpress/testcafe/commit/82a57ef3ba6b374b0fe8a5f91f70e6dd3dcd7548) and the behaviour is the same.

**Comment by ayemelyanenko-chegg** [comment]
I am also seeing this issue

**Comment by Bayheck** [comment]
Hello,
Thank you for reaching out to us.

Could you tell if the issue is similar to [https://github.com/DevExpress/testcafe/issues/8308](https://github.com/DevExpress/testcafe/issues/8308)?

Or do the tests hang but still pass?

**Comment by joshuaegclark-rally** [comment]
Not entirely sure what changed, but I am now having tests hang when running with TestCafe `3.7.0-rc.1` and Chrome ` 130.0.6723.59` on macOS Sonoma 14

The error in question is "Error in Role initializer" "Error: WebSocket is not open: readyState 3 (CLOSED)"

**Comment by ayemelyanenko-chegg** [comment]
As a workaround, if you have Microsoft Edge installed, you can use that for your tests for now as Edge is Chromium based

**Comment by Bayheck** [comment]
Hello,

We have released TestCafe 3.7.0-rc.2, which addresses this issue. Please try it and let us know if the issue is gone.

**Comment by thomas-jakemeyn** [comment]
@Bayheck, problem solved, thank you!
